example/Makefile

```diff
@@ -41,13 +41,17 @@ run:
 	$(HELM) repo add jetstack https://charts.jetstack.io
 	$(HELM) repo update
 	$(HELM) dependency build ../charts/topolvm/
-	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f ./values.yaml
+	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f ./values.yaml --set image.tag=$(TOPOLVM_VERSION)
+	$(KUBECTL) apply -f ./thin-sc.yaml
```

Comment:
We define SCs by values.yaml. Could you update `example/values.yaml`?

Reply:
This is a new storage class, let me see if charts can create multiple storage classes via values.yaml

Reply:
Please refer this file.
https://github.com/cybozu-go/neco-apps/blob/7a43554da869c2aa06c562541402fb16a360aa33/topolvm/base/values.yaml#L80-L134

Reply:
- if I'm updating the new storageclass in `example/values.yaml` this overwrites existing `topolvm-provisioner` sc
- is it fine if I add `topolvm-provisioner` to `example/values.yaml` as well?

Reply:
Yes. Defining `topolvm-provisioner` is needed. This is a bit ugly, though.